DS_1/HighestDegree.java

```diff
@@ -0,0 +1,74 @@
+package datastructure_1;
+import java.util.ArrayList;
+import java.util.Scanner;
+/**
+ * HighestDegree class to find the highest degree of the polynomial variant using the nested list implementation
+ *
+ */
+public class HighestDegree {
+/**
+ * this method create polynomial expression in nested list form
+ * @return Polynomial expression in form of nested list
+ */
+	public ArrayList<ArrayList<String>> createPolynomial()
+	{
+		Scanner sc=new Scanner(System.in);
+		ArrayList<ArrayList<String>> polynomial=new ArrayList<ArrayList<String>>();
+		ArrayList<String> term1=new ArrayList<String>();
+		term1.add("13");
+		term1.add("x");
+		term1.add("2");
+		term1.add("y");
+		term1.add("5");
+		ArrayList<String> term2=new ArrayList<String>();
+		term2.add("13");
+		term2.add("x");
+		term2.add("0");
+		term2.add("y");
+		term2.add("3");
+		ArrayList<String> term3=new ArrayList<String>();
+		term3.add("13");
+		term3.add("x");
+		term3.add("-2");
+		term3.add("y");
+		term3.add("8");
+		polynomial.add(term1);
+		polynomial.add(term2);
+		polynomial.add(term3);
+
+
+		return polynomial;
+	}
+	/**
+	 * 
+	 * @param polynomial is the polynomial expression in form of nested list
+	 * @return the highest degree of the multi variant polynomial 
+	 */
+	public int getHighestDegree(ArrayList<ArrayList<String>> polynomial)
```

Comment:
No need to make method public, if the method is to be used within this file only.